core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -62,8 +63,8 @@
   private int mInAlluxioPercentage;
   private String mUfsFingerprint = Constants.INVALID_UFS_FINGERPRINT;
 
-  private AccessControlList mAcl = AccessControlList.EMPTY_ACL;
-  private DefaultAccessControlList mDefaultAcl = DefaultAccessControlList.EMPTY_DEFAULT_ACL;
+  private AccessControlList mAcl = null;
+  private DefaultAccessControlList mDefaultAcl = null;
```

Comment:
If this can be `null`, the `toString` will throw NPE?

Reply:
I changed it so that the object can never be null, but over the wire, we choose not to serialize it if it is just an empty ACL or defaultACL.